website/content/en/preview/upgrade/upgrade-guide.md

```diff
@@ -1,146 +1,153 @@
-
-## Released Upgrade Notes
-
-### Upgrading to v0.32.0+
+## Upgrading to v0.32.0+ (v1beta1)
 
 * Karpenter now serves the webhook prometheus metrics server on port `8001`. If this port is already in-use on the pod or you are running in `hostNetworking` mode, you may need to change this port value. You can configure this port value through the `WEBHOOK_METRICS_PORT` environment variable or the `webhook.metrics.port` value if installing via Helm.
 * Karpenter now exposes the ability to disable webhooks through the `webhook.enabled=false` value. This value will disable the webhook server and will prevent any permissions, mutating or validating webhook configurations from being deployed to the cluster.
 
-### Upgrading to v0.31.0+
+### v1beta1 Migration
+
+Here is some information you should know about upgrading the Karpenter controller to v0.32:
+
+* **Towards a v1 release**: The latest version of Karpenter sets the stage for Karpenter v1. Karpenter v0.32 implements the Karpenter v1beta1 API spec. The intention is to have v1beta1 be used as the v1 spec, with only minimal changes as needed.
+* **Path to upgrading**: This procedure assumes that you are upgrading from Karpenter v0.31 to v0.32. If you are on an earlier version of Karpenter, review the [Release Upgrade Notes]({{< relref "#release-upgrade-notes" >}}) for breaking changes.
+* **Enhancing and renaming components**: For v1beta1, APIs have been enhanced to improve and solidify Karpenter APIS, as well as to make features more aligned with other Kubernetes autoscaling technologies. Part of these enhancements includes renaming Kind elements Provisioner (now NodePool), Machine (now NodeClaim), and AWSNodeTemplate (now EC2NodeClass). To fully transition to v1beta1, you need to update those Kind elements in your manifests and make other changes to your cluster, as described later.
+* **Running v1alpha1 alongside v1beta1**: Having different Kind names for v1alpha5 and v1beta1 allows them to coexist for the same Karpenter controller for v0.32. This gives you time to transition to the new v1beta1 APIs while existing Provisioners and other objects stay in place. Keep in mind that there is no guarantee that the two versions will be able to coexist in future Karpenter versions.
+
+Some things that will help you with this upgrade include:
+
+* **[Karpenter Upgrade Reference]({{< relref "upgrade-ref.md" >}})**: Provides a complete reference to help you transition your Provisioner, Machine, and AWSNodeTemplate manifests, as well as other components, to be able to work with the new v1beta1 names, labels, and other elements.
+* **[Karpenter conversion tool](https://github.com/aws/karpenter/tree/main/tools/karpenter-convert)**: Simplifies the creation of NodePool and EC2NodeClass manifests.
```

Comment:
Since installation procedures for third-party tools can change, we usually just point to those procedures. But since it's only one line, I added it to a new Prerequisites section to save people the click.

Reply:
Okay. I added a prerequisites procedure at the beginning of the procedure for installing karpenter-convert.